src/patternfly/components/Page/page.scss

```diff
@@ -76,6 +78,8 @@ $pf-c-page--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg"
   --pf-c-page__sidebar--m-light--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
   --pf-c-page__sidebar--m-light--BorderRightWidth: var(--pf-global--BorderWidth--sm);
   --pf-c-page__sidebar--m-light--BorderRightColor: var(--pf-global--BorderColor--100);
+  --pf-c-page__sidebar--m-light--c-divider--after--BackgroundColor: var(--pf-global--BorderColor--300);
```

Comment:
@mcarrano @mceledonia @mmenestr - do we need to continue to support the light theme vertical nav with v5 or is that something we could drop? https://www.patternfly.org/v4/components/navigation/html#default-in-light-mode